src/Microsoft.Health.Dicom.Api/Controllers/RetrieveController.cs

```diff
@@ -152,6 +153,30 @@ public async Task<IActionResult> GetSeriesMetadataAsync([FromHeader(Name = IfNon
         return CreateResult(response);
     }
 
+    [Produces(KnownContentTypes.ImageJpeg, KnownContentTypes.ImagePng)]
+    [ProducesResponseType((int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NoContent)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotAcceptable)]
+    [HttpGet]
+    [VersionedPartitionRoute(KnownRoutes.InstanceRenderedRoute)]
+    [VersionedRoute(KnownRoutes.InstanceRenderedRoute)]
+    [AuditEventType(AuditEventSubType.RetrieveRendered)]
+    public async Task<IActionResult> GetRenderedInstanceAsync(
+        string studyInstanceUid,
+        string seriesInstanceUid,
+        string sopInstanceUid,
+        [FromQuery] int quality = 100)
```

Comment:
Quality is specific to jpeg right? Should we rename to be specific

Reply:
do you mean changing the name of the actual query paramater or just in our code? The actual query param is defined as `quality` in the dicom spec so I was keeping it consistent

Reply:
no just in code, quality is fine as a param